src/components/profile/ProfilePictureSection.vue

```diff
@@ -0,0 +1,205 @@
+                }
+                busy.value = false;
+            }
+
+            const pictureChanged = computed(() => {
+                const value = selectedPicture.value !== fallbackPicture.value;
+                return value;
+            });
+
+            function triggerFileUpload() {
+                (document.getElementById("uploadFile") as any).click();
+            }
+
+            function uploadPicture(e: Event) {
+                const files: FileList | null = (e.target as HTMLInputElement)?.files;
+                if (files == null) return;
+
+                const file = files[0];
+                fileToUpload = file;
+                const reader = new FileReader();
+                reader.readAsDataURL(file);
+                reader.onload = (e) => {
+                    selectedPicture.value = e.target?.result;
+                };
+            }
+
+            async function confirmPicture() {
+                busy.value = true;
+                const userManagement = new UserManagement();
+                const response = await userManagement.updateProfilePicture(username, fileToUpload);
+                const handler = new ProfilePictureUpdateResponseHandler();
+                const result = await handler.handleResponse(response);
+                if (result) {
+                    fallbackPicture.value = selectedPicture.value;
+                    errorBag.value = new ErrorBag();
+                } else {
+                    errorBag.value = new ErrorBag(handler.errorList);
+                    console.log("Error: Uploading profile picture failed!");
+                }
+                busy.value = false;
+            }
+
+            function resetPicture() {
+                selectedPicture.value = fallbackPicture.value;
+            }
+
+            async function confirmDeletePicture() {
+                let modal = deletePictureModal.value;
+                let action = modal.action;
+                modal.show().then((response: typeof action) => {
+                    switch (response) {
+                        case action.CANCEL: {
+                            //do nothing
+                            break;
+                        }
+                        case action.DELETE: {
+                            deleteProfilePicture();
+                            break;
+                        }
+                    }
+                });
+            }
+
+            async function deleteProfilePicture() {
+                busy.value = true;
+                const userManagement = new UserManagement();
+                const response = await userManagement.deleteProfilePicture(username);
+                const handler = new GenericResponseHandler();
+                const result = await handler.handleResponse(response);
+                if (result) {
+                    getProfilePicture();
+                } else {
+                    console.log("Error: Picture Deletion Failed!");
```

Comment:
Either delete log also or use
```suggestion
                    console.error("Picture Deletion Failed!");
```